ClosedXML/Excel/Ranges/IXLRangeBase.cs

```diff
@@ -93,6 +94,11 @@ public interface IXLRangeBase : IDisposable
 
         IXLCells CellsUsed(Boolean includeFormats, Func<IXLCell, Boolean> predicate);
 
+        /// <summary>
```

Comment:
Why does this need to be publically exposed?

Reply:
Tried to avoid explicit typecasting in ```CellRangeReference``` and thought it may be useful to have a possibility to quickly get all values in a range no matter how big it is. I do not insist on making this public, however.

Reply:
Yes, I'd prefer to keep the interface unchanged unless we deliberately want to expose them as public.